src/pint/models/dispersion_model.py

```diff
@@ -320,42 +320,80 @@ def __init__(self):
                 description="Dispersion measure",
             )
         )
+
+        self.add_DMX_range(None, None, dmx=0, frozen=False, index=1)
```

Comment:
Is it necessary to add this spurious DMX range?

Reply:
For now, yes. Since the model builder checks the existing parameters. We can come back when the model builder gets updated.

Reply:
Can it be removed automatically, during setup() say?

Reply:
I think that removing it in setup will break the model builder. also, because i biffed the commit stream here, i opened a new PR (https://github.com/nanograv/PINT/pull/910).